### ISSUE TITLE
How to use ROC-AUC score for Biobert multi-label text classification?

### ISSUE THREAD

**Issue by miladnouriezade** [amaiya/ktrain#166]
Hi, I've used `text. Transformer` module to train a multi-label classifier, How can I use the ROC-AUC score to evaluate my model?
I would be grateful if you could help me.

```bash
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)
MODEL_NAME = 'monologg/biobert_v1.1_pubmed'
t = text.Transformer(MODEL_NAME, maxlen=500, class_names=['......'])
trn = t.preprocess_train(X_train, y_train)
val = t.preprocess_test(X_test, y_test)
model = t.get_classifier(multilabel=True)
learner = ktrain.get_learner(model, train_data=trn, val_data=val, batch_size=8)
learner.fit(5e-5,n_cycles=4, cycle_len=1, cycle_mult=2, early_stopping=3)
```

**Comment by Chiuchiyin** [comment]
Thank you. This helped me with my project.
